### DISCUSSION TITLE
Deduplication in Grafana Mimir

### DISCUSSION THREAD

**Discussion by rosh0450** [grafana/mimir#7268] [Help and support] [answered]
I have 2 Prometheus instance (for HA) remote writing same metrics to Grafana Mimir. 
 Does Mimir has some provisions to deduplicate these metrics ??

 I went through the docs here: https://grafana.com/docs/mimir/latest/configure/configure-high-availability-deduplication/#configure-grafana-mimir-high-availability-deduplication
 
 But this talks about the case when Mimir scrapes metrics from Prometheus not the case when Prometheus remote write to Mimir.

Thanks in advance.

**Comment by pstibrany** [comment, marked as answer]
Hello,

Mimir cannot scrape metrics directly, it can only receive metrics from Prometheus or other sources. The link you posted describes exactly the same scenario as you are — multiple Prometheus servers writing same metrics to Grafana Mimir.
